README.md

```diff
@@ -472,16 +589,16 @@ async def main() -> None:
 
         systems = await simplisafe.get_systems()
         for system_id, system in systems.items():
-            # Return the current access token:
+            # Return the current access token: (higly advisable to not show on your screen; hackers can install software that runs in the background, takes screenshots, and send them to their computer.)
```

Comment:
This isn't necessary considering the paragraph that opens this section already gives a warning:

>VERY IMPORTANT NOTE: the API object contains references to SimpliSafe™ access and refresh tokens. It is vitally important that you do not let these tokens leave your control. If exposed, savvy attackers could use them to view and alter your system's state. You have been warned; proper usage of these properties is solely your responsibility.

Reply:
Sorry; I missed that. 😁 How do I remove that?

Reply:
Submit a new commit to your PR that removes your changes to this section.

Reply:
Done that.